os-appium-tests/tests/helpers/NativeAlert.ts

```diff
@@ -0,0 +1,62 @@
+import {DEFAULT_TIMEOUT, DEFAULT_TIMEOUT_INTERVAL} from '../constants';
+
+const SELECTORS = {
+    ANDROID: {
+        ALERT_BUTTON: '*//android.widget.Button[@text="{BUTTON_TEXT}"]',
+        ALERT_DIALOG: '*//android.widget.LinearLayout[@resource-id="com.android.packageinstaller:id/dialog_container"]', // Sandra
```

Comment:
Can you clean the leftover comments from the project? `// Sandra` is not quite informative :)